src/source/source_cache.js

```diff
@@ -475,6 +477,7 @@ class SourceCache extends Evented {
             clearTimeout(this._timers[id]);
             this._timers[id] = undefined;
         }
+        // TODO should we keep this??
         this._source.fire('data', { tile: tile, coord: tile.coord, dataType: 'tile' });
```

Comment:
I agree that this event isn't useful -- if we're not using it anywhere internally, then I'm all for removing it.

What _might_ be useful someday in the future would be a way for users to listen for changes to the list of currently renderable/rendered tile coordinates -- but this doesn't really provide that, and anyway that's a super non-urgent nice-to-have.